src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.SourceGenerators.targets

```diff
@@ -19,6 +19,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_RazorSdkToolsDirectoryRoot>$(RazorSdkDirectoryRoot)\tools\</_RazorSdkToolsDirectoryRoot>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <RazorEncConfigFile>$(_RazorSdkSourceGeneratorDirectoryRoot)RazorSourceGenerator.razorencconfig</RazorEncConfigFile>
+    </PropertyGroup>
+
+    <EditorConfigFiles Include="$(RazorEncConfigFile)" Condition="'$(DesignTimeBuild)' == 'true'"/>
```

Comment:
@pranavkm I don't think we need to do a building in VS check for this case but let me know if my assumption is incorrect.

Reply:
It might not hurt to toss that in for good measure.